pkg/rendering/renderer.go

```diff
@@ -321,6 +329,9 @@ func injectValuesOverrides(values *Values, mch *v1.MultiClusterHub, images map[s
 
 	values.HubConfig.ClusterSTSEnabled = isSTSEnabled
 
+	// TODO: check if the operator is running on OCP cluster
+	values.Global.DeployOnOCP = "true"
```

Comment:
Is there a reason why this is a string and not a boolean?

Reply:
nope.  boolean is better than string here.
changed to boolean.